android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -704,6 +704,15 @@ public void fetchNativeSdkInfo(Promise promise) {
         }
     }
 
+    public void fetchNativePackageName(Promise promise) {
+        promise.resolve(packageInfo.packageName);
+    }
+
+    public void fetchNativeStackFramesBy(Promise promise) {
+        logger.log(SentryLevel.ERROR, "This method works only on iOS.");
```

Comment:
Can we actually not call this method at all? so the log isn't needed and we spare 1L, a smaller bundle size.

Reply:
It's not called but we have to implement at least an empty method. Make sense, I'll remove the code to make the bundle smaller.